plugins/travis-ci/data.txt

```diff
@@ -0,0 +1 @@
+ TRAVIS CI TOKEN wo90eVG0Htjh0wksszV1zA
```

Comment:
I can't find a place where this file is used. Can it be deleted?

Reply:
sure, I've created it only for testing to keep my testing token near at hand. I'll remove it as soon as I finish with plugin